spec/worker/gamedev_worker_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe GamedevWorker do  
+  it 'enqueues a Gamedev job' do
+    Sidekiq::Testing.fake!
+    expect {
+      GamedevWorker.perform_async
+    }.to change(GamedevWorker.jobs, :size).by(1)
+  end
+
+  context 'with succesful response' do
+    it 'creates external posts' do
+      VCR.use_cassette('questions') do
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.